extensions/amp-link-rewriter/validator-amp-link-rewriter.protoascii

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+tags: {  # amp-link-rewriter
```

Comment:
Did we check that only one `<amp-link-rewriter>` is allowed. cc @honeybadgerdontcare

Reply:
Newer tags do not `allow_duplicates`, only grandfathered ones. But your question is about `<amp-link-rewriter>` and not necessarily the script tag to include it. For there to be only one `<amp-link-rewriter>` tag then on line 49 or after there needs to be a `unique: true` to make sure there can only be one per document.

Reply:
added `unique: true` and additional tests